### DISCUSSION TITLE
Problem opening software MACOS with AIR runtime - version 33.1.1.444

### DISCUSSION THREAD

**Discussion by Pennylane499** [airsdk/Adobe-Runtime-Support#790] [Q&A]
I am trying to open a software, namely 'Cambridge Objective Proficiency', which states that I have to install Air in order to execute the program. I have a MAC Catalina version 10.15.4 (19E287) and whenever I try to execute the program I just come across a white screen without a way to go further. Any ideas?
[Installation instructions READ ME.pdf](https://github.com/Gamua/Adobe-Runtime-Support/files/6311885/Installation.instructions.READ.ME.pdf)

**Comment by ajwfrost** [comment]
If you're using Catalina, can you try installing the AIR runtime from the below link before you try running the 'Cambridge Objective Proficiency' installer? (I'm curious, what format does this installer come as? is it dmg, pkg, etc? and if dmg, then what's inside..?)
https://airsdk.harman.com/runtime


thanks

**Reply by Pennylane499** [reply]
![Captura de pantalla 2021-04-14 a las 17 59 38](https://user-images.githubusercontent.com/82527042/114741810-6eaef580-9d4b-11eb-8fc1-6cd23627dee3.png)

I am honestly confused myself, it is neither dmg nor pkg from what I can see. The app that you see in the image was inside a zip folder and has the name of  'Objective Proficiency.air'

**Reply by ajwfrost** [reply]
Okay just a zip, that's fine. So yes, if you have a ".air" file then it requires the AIR runtime to open it. From the looks of that screenshot, you already have the AIR runtime installed .. so what happens when you double-click the .air file?

If this is what it's saying is damaged, then could you please check in Finder under /Applications, if you click on the "Adobe AIR Installer" application, what version does it give you? (even better if you show contents, and look at the Contents/Info.plist file, there should be a full version code in there under CFBundleVersion).

thanks

**Reply by Pennylane499** [reply]
Yeah, everything is installed correctly, the only problem is when I get to open the software, the screen just remains white without showing any of the contents of the software. 
It is version 33.1 and the contents of the application
 you may see in the screenshot. 
![Captura de pantalla 2021-04-16 a las 7 48 32](https://user-images.githubusercontent.com/82527042/114977932-cd2dbe00-9e88-11eb-950c-507529f713aa.png)
